src/Pages/Login/login.scss

```
@@ -0,0 +1,153 @@
+@import '../../Styles/common.scss';
+@import '../../Styles/reset.scss';
+* {
+  box-sizing: border-box;
+}
+.login {
+  width: 40vw;
+  height: 100%;
+  margin: 0 auto;
+  .loginTitle {
+    width: 100%;
+    height: 30vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    h1 {
+      font-size: 1.5rem;
+      font-weight: 900;
+      padding: 1rem;
+    }
+    p {
+      font-size: 5px;
+      font-weight: 500;
+    }
+  }
+  .loginForm {
+    .memberWrap {
+      height: 4rem;
+      font-weight: 900;
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      text-align: center;
+      line-height: 3.8rem;
+      #hpointMember {
+        border-left: 1px solid #bebebe;
+        border-top: 1px solid #bebebe;
+      }
+      #greatingMember {
+        border-right: 1px solid #bebebe;
+        border-top: 1px solid #bebebe;
+      }
+    }
+    form {
+      padding-top: 1rem;
+    }
+    input {
+      display: flex;
+      flex-direction: column;
+      margin: 10px auto;
+      width: 100%;
+      height: 3rem;
+    }
```

```suggestion
    input {
      margin: 10px auto;
      width: 100%;
      height: 3rem;
    }
```
input 태그는 자식 요소를 가질 수 없으므로 flex 속성은 부적절해보입니다

---

> 고생하셨습니다! 코멘트 확인해주시고 모르겠는 부분은 꼭 재질문 해주세요! :)

> 이해가 안되는 부분을 제외하고 전부 수정하였습니다!